core/jazz_ui/build.website

```diff
@@ -0,0 +1,61 @@
+#
+# TODO: Consider moving jq → config
+#
+JQ_BINARY=https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+S3_BUCKET={swagger_bucket}.s3.amazonaws.com
+TEMP=$(mktemp -d)
+
+#
+# Pull paths from S3 bucket manifest
+#
+echo -e '\nPulling paths from S3 bucket manifest...\n'
+
+curl -L $S3_BUCKET > $TEMP/1
+xpath -e '/ListBucketResult/Contents/Key/text()' $TEMP/1 > $TEMP/2
+paths=$(sed '/^jazz\/.*swagger.json/!d; /-dev/d; /stg/d' $TEMP/2)
+
+echo -e '\nDone!\n'
+echo '{"swagger": "2.0"}' > $TEMP/1
+#
+# End pull paths
+#
+
+#
+# Download and merge jazz/* paths into placeholder JSON document
+#
+echo -e '\Downloading jazz/* paths and merging JSON...\n'
```

Comment:
- Remove `x-amazon-apigateway-integration` from each swagger (if exists)
- Remove OPTIONS method for all APIs (because they don’t add value when browsing API docs)

Reply:
Addressed.